Gemfile.lock

```diff
@@ -1036,4 +1036,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.4.19
+   2.5.23
```

Comment:
@jayjay-w should this be here?

Reply:
Thank you for removing that, Caio, it was a supposed to be deleted in this commit: https://github.com/meedan/check-api/pull/2195/commits/207f2462e5669044c8839aa274dafce474b00434